http.go

```diff
@@ -76,8 +76,13 @@ func probeHTTP(target string, w http.ResponseWriter, module Module) (success boo
 	if config.Method == "" {
 		config.Method = "GET"
 	}
+	if config.Path == "" {
+		config.Path = "/"
+	}
+
+	log.Infof("probeHTTP to %s%s", target, config.Path)
```

Comment:
I consider anything more than 1-2 info logs a minute spammy as it'll hide the interesting logs. This information is primarily useful for debugging, there's no point filling up the logs with it during normal operations.